deployments/charts/kuma/values.yaml

```diff
@@ -170,3 +174,8 @@ ingress:
     annotations: {}
     # -- Port on which Ingress is exposed
     port: 10001
+
+kumactl:
+  image:
+    # -- The kumactl image repository
+    repository: kumactl
```

Comment:
Hmm is this `repository` for real? How do we call it for the other images?

Reply:
Yes it is, there is an example for other ones:
```yaml
dataPlane:
  image:
    # -- The Kuma DP image repository
    repository: "kuma-dp"
    # -- Kuma DP ImagePullPolicy
    pullPolicy: IfNotPresent

  initImage:
    # -- The Kuma DP init image repository
    repository: "kuma-init"
```

It has to be in this format as I'm using our helper
```
{{/*
params: { image: { registry?, repository, tag? }, root: $ }
returns: formatted image string
*/}}
{{- define "kuma.formatImage" -}}
{{- $img := .image }}
{{- $root := .root }}
{{- $registry := ($img.registry | default $root.Values.global.image.registry) -}}
{{- $repo := ($img.repository | required "Must specify image repository") -}}
{{- $defaultTag := ($root.Values.global.image.tag | default $root.Chart.AppVersion) -}}
{{- $tag := ($img.tag | default $defaultTag) -}}
{{- printf "%s/%s:%s" $registry $repo $tag -}}
{{- end -}}
```